manager_test_helper.go

```diff
@@ -147,6 +148,23 @@ var TestManagerPolicies = []*DefaultPolicy{
 			"owner": &EqualsSubjectCondition{},
 		},
 	},
+	//Two new policies which do not persist in MySQL correctly
```

Comment:
So the tests might be a bit confusing here... Anything that is in `TestManagerPolicies` will be added automatically to the manager that is currently being tested

Reply:
Alright, two questions. I changed the assert, so the assert is not failing now, but : 
* Why is``` go test ./... ```not working? I am not able to test ```manager_test_helper.go``` file locally.
* If the policies I added in TestManagerPolicies above are automatically added to the manager that is currently being tested, why are the tests failing now? ```ladon.Conditions(nil)``` is mentioned in the error trace, but I do not understand.

Reply:
> Why is `go test ./...` not working? I am not able to test manager_test_helper.go file locally.

I don't know, make sure to run it from the project root. Add a test that fails for sure (`assert.True(false)`) and see if tests fail.

> If the policies I added in TestManagerPolicies above are automatically added to the manager that is currently being tested, why are the tests failing now? ladon.Conditions(nil) is mentioned in the error trace, but I do not understand.

You omitted the conditions which causes them to be of nil value but it the manager never returns conditions of type nil in order to prevent invalid pointer panics. If you look at the other policies you'll notice that each policy sets `Conditions: Conditions{},` which you omitted